### PR TITLE
Add vb.net support

### PR DESCRIPTION
**Pull request by Vidarls**
Rough support for Vb.net projects

### CODE REVIEW

**Comment by forki** [comment]
Wow. Pretty cool stuff

**Comment by Vidarls** [comment]
Hit a case when using InteropFormsToolkit (http://www.microsoft.com/en-us/download/details.aspx?id=3264) that in its official form only supports vb.net projects.

File: build.template [hunk 1]
```diff
@@ -163,6 +169,17 @@ Target "SourceLink" (fun _ ->
         proj.CreateSrcSrv baseUrl repo.Revision (repo.Paths files)
         Pdbstr.exec proj.OutputFilePdb proj.OutputFilePdbSrcSrv
     )
+    //VB projects
+    !! "src/**/*.vbproj"
```
Comment (forki) [inline build.template]: I'd love to see this boilerplate scraped. ;-)

**Comment by Vidarls** [comment]
This one should be ok now as well

**Comment by forki** [comment]
awesome!

**Pull request merged**